From 32f69c2caa1f933173c8e05cb5fab66ac07b31c5 Mon Sep 17 00:00:00 2001
From: Sean Cross <xobs@kosagi.com>
Date: Wed, 10 Feb 2016 18:08:55 +0800
Subject: [PATCH 48/65] package-kerne: update to v4.4-novena-r6

This new version includes some fixes for audio on resume, and a hack
to try and increase stability on the LDB on resume.

Signed-off-by: Sean Cross <xobs@kosagi.com>
---
 package-kernel.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package-kernel.sh b/package-kernel.sh
index ed9c655..7477886 100755
--- a/package-kernel.sh
+++ b/package-kernel.sh
@@ -1,7 +1,7 @@
 #!/bin/sh
 # A script to build the kernel .deb files, as used by Kosagi.
 threads=$(($(grep '^processor' /proc/cpuinfo  | cut -d: -f2 | tail -n1)+1))
-version=4.4-novena-r4
+version=4.4-novena-r6
 
 if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
 	echo "Repo has local changes.  Stash or commit them."
-- 
2.7.3

